docs/product-getting-started-guide/src/main/asciidoc/data-object-violation-create-proc.adoc

```diff
@@ -0,0 +1,79 @@
+[id='data-object-violation-proc']
+= Creating the Violation data object
+
+The following chapter instructs you how to create the Violation data object containing data fields, such as: Violation Date, Fine Amount, and Speed Limit.
+
+[float]
+== Prerequisites
+
+You have created the Driver department traffic violations project.
+
+.Procedure
+. Log in to {CENTRAL}. Click *Menu* -> *Design* -> *Projects*, then *Driver department traffic violations*.
+. Click *Create New Asset* -> *Data Object*.
+. In the *Create new Data Object* wizard, enter the following values:
+.. *Data Object*: `Violation`.
+.. *Package*: select `org.jboss.example.traffic_violations`.
+. Click *+Ok*.
++
+
+.Create new Data Object window
+image::data-object.png[]
+
+== Adding the Violation data object data fields
```

Comment:
I'll probably only mention this here, since it's an issue throughout the doc that you're fully aware of. I know you said you don't want to add intro statements and .Procedure with every task because you feel like it clutters. Up to you. IMO, I don't think so, and I think the user would actually benefit greatly from it. It'd be quite easy and very helpful to add one here, for context. Large payoff for the user for little writer effort.

Example (my whack at it):

Adding the Violation data object data fields
"The data fields in the `Violation` data object define the available constraints that you can select from when you define your violation rules later."

".Procedure"
(steps) ....

This is an important and very helpful consideration that the user probably doesn't realize. Otherwise, he/she is just following the task blindly, not knowing why it's necessary.

Reply:
I fully agree and have done this for each section today.

Reply:
add to that.. I totally replaced what I had added with your suggestion. No pride here:)